source/style-guide/writing/write-to-the-user.txt

```diff
@@ -82,11 +113,14 @@ talk to users directly by using *second person*, addressing the user as
 .. _subjunctive-mood:
 
 Avoid the Subjunctive Mood
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+--------------------------
 
 The subjunctive mood expresses doubt. *That doesn’t engender trust from
-readers about our content!* Avoid words like “should” and “could”; they
-imply uncertainty.
+readers about our content!* Avoid words that imply uncertainty:
+
+- should
+- could
+- would
```

Comment:
Not sure if we hear about this elsewhere, but this might be a good place to talk about must, must not, may, may not, etc. The MDB specs (like [this one](https://github.com/mongodb/specifications/blob/master/source/auth/auth.rst#mongodb-iam)) talk about these with the following language:

> META
> The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.

And here's the RFC: https://www.ietf.org/rfc/rfc2119.txt

Short and sweet!

I feel like you address this later, so ignore this if I just haven't gotten to it yet.

Reply:
Would like to, but our language is a little more specific.